lib/otp/views/configure_yubiotp_dialog.dart

```diff
@@ -390,6 +384,28 @@ class _ConfigureYubiOtpDialogState
     );
   }
 
+  RichText _createUploadText(BuildContext context, AppLocalizations l10n) {
+    final uploadText = l10n.l_exported_can_be_uploaded_at(uploadOtpUri.host);
+    final host = uploadOtpUri.host;
+    final parts = uploadText.split(RegExp('(?=$host)|(?<=$host)'));
+
+    return RichText(
+      text: TextSpan(
+        style: Theme.of(context)
+            .textTheme
+            .bodySmall
+            ?.copyWith(color: Theme.of(context).colorScheme.onSurfaceVariant),
+        children: [
+          ...parts.map(
+            (e) => e == uploadOtpUri.host
+                ? _createUploadOtpLink(context)
+                : TextSpan(text: e),
+          )
+        ],
+      ),
+    );
+  }
+
   TextSpan _createUploadOtpLink(BuildContext context) {
     final theme = Theme.of(context);
     final uploadOtpUri = Uri.parse('https://upload.yubico.com');
```

Comment:
Looks like this is now duplicated above?

Reply:
Good catch! Will remove that.